pandas/tests/io/pytables/test_store.py

```diff
@@ -1,6 +1,8 @@
-import datetime
+import datetime as dt
+from datetime import datetime
```

Comment:
there shouldn't be any need for both, just use `dt.datetime`

Reply:
Removed second import and used dt.datetime.  I renamed variable dt to mydt.